docs/resources/route_control_profile.md

```diff
@@ -7,50 +10,159 @@ description: |-
+
+## Schema ##
+
+### Required ###
+
+* `parent_dn` - (string) The distinguished name (DN) of the parent object, possible resources:
+  - [aci_tenant](https://registry.terraform.io/providers/CiscoDevNet/aci/latest/docs/resources/tenant) ([fvTenant](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvTenant/overview))
+  - [aci_l3_outside](https://registry.terraform.io/providers/CiscoDevNet/aci/latest/docs/resources/l3_outside) ([l3extOut](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/l3extOut/overview))
+* `name` (name) - (string) The name of the Route Control Profile object.
 
-- `parent_dn` - (Required) Distinguished name of the parent object.
-- `name` - (Required) Name of router control profile object.
-- `annotation` - (Optional) Annotation for router control profile object.
-- `description` - (Optional) Description for router control profile object.
-- `name_alias` - (Optional) Name alias for router control profile object.
-- `route_control_profile_type` - (Optional) Component type for router control profile object. Allowed values are "combinable" and "global". Default value is "combinable".
+### Read-Only ###
 
-## Attribute Reference
+* `id` - (string) The distinguished name (DN) of the Route Control Profile object.
 
-The only attribute that this resource exports is the `id`, which is set to the Dn of the Route Control Profile.
+### Optional ###
+  
+* `annotation` (annotation) - (string) The annotation of the Route Control Profile object.
+  - Default: `orchestrator:terraform`
+* `route_map_continue` (autoContinue) - (string) The route_map_continue applies a continue statement for all user-configured sequences (contexts) in a given BGP route profile so that switches will continue to find matches with subsequent user-configured sequences in the route map. Without the continue statement, after a route matches one of the sequences in a route map, switches will not continue to check other sequences.
+  - Default: `no`
+  - Valid Values: `no`, `yes`.
+* `description` (descr) - (string) The description of the Route Control Profile object.
+* `name_alias` (nameAlias) - (string) The name alias of the Route Control Profile object.
+* `owner_key` (ownerKey) - (string) The key for enabling clients to own their data for entity correlation.
+* `owner_tag` (ownerTag) - (string) A tag for enabling clients to add their own data. For example, to indicate who created this object.
+* `route_control_profile_type` (type) - (string) When the type is Match Prefix and Routing Policy (combinable), it is the default value. Pervasive subnets (fvSubnet) and external subnets (l3extSubnet) are combined with a route profile and merged into a single route map (or route map entry). To use an explicit prefix list, the type should be set to Match Routing Policy Only (global). The route profile is the only source of information to generate a route map, and it will overwrite other policy attributes.
```

Comment:
do we need "When the type is Match Prefix and Routing Policy (combinable), it is the default value", when default is already mentioned below. Is this part "Pervasive subnets (fvSubnet) and external subnets (l3extSubnet) are combined" about combinable? Perhaps do the same as you do for global?